resources/donations.md

```diff
@@ -23,36 +23,63 @@ The response from the `donations` endpoint is an array of donation objects.
   {
     "displayName": "Friendly Donor",
     "participantID": 4024,
-    "amount": 20,
+    "amount": 10,
     "donorID": "EB8610A3FC435D58",
     "avatarImageURL": "https://static.donordrive.com/clients/try/img/avatar-constituent-default.gif",
     "createdDateUTC": "2019-10-30T18:01:18.513+0000",
     "eventID": 581,
     "teamID": 5074,
-    "donationID": "DF4E676D0828A8D5"
+    "donationID": "DF4E676D0828A8D5",
+	"links": {
```

Comment:
can you please check indentation on your new properties?

Reply:
it was tabs vs spaces. they're all fixed now